mobile/lib/features/trade/domain/price.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:get_10101/bridge_generated/bridge_definitions.dart' as bridge;
+
+import '../../../common/dummy_values.dart';
+
+class Price {
+  final double bid;
+  final double ask;
+
+  Price({required this.bid, required this.ask});
+
+  static Price fromApi(bridge.BestPrice bestPrice) {
+    return Price(
+        bid: bestPrice.bid ?? dummyBidPrice,
```

Comment:
so far I moved things around into a dedicated file and put a comment there - that in the prototype we have many dedicated values (with a TODO to remove them). 

would it be helpful to say that we should really be able to depict that there might be no order? (either during startup, but also, there can be a situation that order book has no orders)

Reply:
hmmm.. I guess its fair to say that the app should eventually be able to handle a scenario where there are no orders, or the orderbook is not reachable.

Reply:
exactly! I have made the backend code support that, because it was easy to do - frontend turned out to be already too complex for that, so I have taken the shortcut.
I will add a comment there to make it more clear, it won't hurt 😇